tox.ini

```diff
@@ -21,8 +25,12 @@ passenv =
     AWS_SECRET_ACCESS_KEY
     GCS_TEST_FILE_URI
     GCS_PROJECT_ID
+#allow tox virtualenv to upgrade pip/wheel/setuptools
+download = true 
 commands =
     py.test --cov=scrapy --cov-report= {posargs:--durations=10 docs scrapy tests}
+install_command =
+    pip install --use-feature=2020-resolver {opts} {packages}
```

Comment:
Are we testing only with the new resolver now? 
Can you see any cases where new resolver works, but the old one doesn't?

Reply:
I don’t think we can discard that scenario. I think it’s possible that the new resolver will fix issues that the old one does not.

However, I’m not sure which jobs should get the new resolver. My original plan was to just set it on 1 job, e.g. extra-deps (or a new job also using the development version of Python, but that would probably be too unstable). 

On the other hand, I think pinned environments are the ones where a resolver may have a higher impact, so I wonder if instead we should use the new resolver in half the pinned environments. Also, given https://github.com/scrapy/scrapy/pull/4814#issuecomment-706230011, I think it would also be fair to say that we must use the new resolver in *all* pinned environments.

Reply:
I'm not sure what's the best option @Gallaecio, but it looks like running new resolver only on some jobs (not on all of them) would be better. Probably it is not worth it to have a separate env just to test env a different resolver; I'm worried about CI runtime creep.

Reply:
I’m going to restrict it to extra-deps for now. As soon as the tests finish with that change, I think we should check the pinned-dependencies jobs for [cases where a newer Twisted is actually installed](https://github.com/scrapy/scrapy/pull/4814#issuecomment-706230011), and also switch those to the new resolver.

Reply:
Done.